src/components/Attachment/hooks/useAudioController.ts

```diff
@@ -56,8 +111,8 @@ export const useAudioController = ({
       const currentTime = ratio * audioRef.current.duration;
       setSecondsElapsed(currentTime);
       audioRef.current.currentTime = currentTime;
-    },
-    [],
+    }, 60),
```

Comment:
Having this function throttled means that the `secondsElapsed` value only updates about 16 times per seconds. That means that the dragging interaction on the progress bar will feel laggy.

Do you think it's possible to either:
1. Separate the displayed progress (which will update optimistically) and the actual progress (which will update with a throttle)
2. Reduce the throttle duration to at least 16ms (for 60fps)
3. Or just remove throttling here?

Reply:
I decided to set it to 60Hz :)